Automattic/jetpack
# VideoPress: Update poster frame by playing preview video

Fixes #29767

## Proposed changes:

* Updates the poster frame by playing the preview video

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Add a VideoPress video block
* Go to the `Poster and preview` panel
* Select  `Pick from video frame`
* Click on the preview video and check that the video frame value is updated as the video plays
* Click the video again to stop at the desired timestamp
* Save the post and wait for the poster image to be generated
* Reload the page
* Check that the selected timestamp is persisted
* Check `Video preview on hover`
* Check that changing the poster frame timestamp does not interfere with the preview on hover controls

https://user-images.githubusercontent.com/8486249/231837875-7e6f1f08-bd3c-476e-b8e5-3c3adae4104e.mp4
